src/components/Persons.vue

```diff
@@ -0,0 +1,16 @@
+<template>
+  <v-container fluid>
+    <v-select
+      prepend-icon="mdi-account-multiple"
+      :items="items"
+      label="Number of Members"
+    ></v-select>
```

Comment:
스티디 인원 수는 select로 고르게 하는 것보다는 text-field로 숫자만 입력하게 하는 건 어떨까요? 아래처럼 만들면 숫자만 입력받을 수 있거든요.

```suggestion
    <v-text-field
      prepend-icon="mdi-account-multiple"
      label="Number of Members"
      type="number"
    ></v-text-field>
```

Reply:
동영님 말처럼 text-field로 숫자만 입력하는게 좋을 것 같네요 ~ 수정하겠습니다:)